webapp/channels/src/components/login/login.tsx

```diff
@@ -460,6 +460,12 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
         };
     }, []);
 
+    useEffect(() => {
+        if (hasError) {
+            loginIdInput.current?.focus();
```

Comment:
I see Level Access said we should move focus back to the invalid input, but I'm not sure if a user would expect that. I can't find an example of validation errors in the ARIA APG, but I did find [this example in the W3C docs which doesn't do that](https://www.w3.org/WAI/tutorials/forms/notifications/). Googling it more, I see a lot of people asking how to implement it, but there's at least a few posts where people are recommending against it, especially when forms are validation on blur where it'd really screw someone up.

In this case, we're only validating the inputs on submit, so this makes sense even if I'm not sure that will work everywhere